packages/react/src/component-tests/IcLoadingIndicator/IcLoadingIndicator.cy.tsx

```diff
@@ -0,0 +1,272 @@
+  beforeEach(() => {
+    cy.injectAxe();
+  });
+
+  afterEach(() => {
+    cy.task("generateReport");
+  });
+
+  it("renders", () => {
+    mount(<IndeterminateCircularLoadingIndWithLabel />);
+    cy.findShadowEl(
+      "ic-loading-indicator",
+      "ic-typography#ic-loading-label"
+    ).should(HAVE_TEXT, "Loading...");
+
+    cy.checkA11yWithWait();
+  });
+
+  it("renders determinate", () => {
+    mount(
+      <DeterminateCircularLoadingIndWithLabel
+        progress={defaultArgs.progress}
+        max={defaultArgs.max}
+        min={defaultArgs.min}
+      />
+    );
+    cy.findShadowEl(
+      "ic-loading-indicator",
+      "ic-typography#ic-loading-label"
+    ).should(HAVE_TEXT, "Loading...");
+
+    cy.get("ic-loading-indicator").invoke("prop", "progress").should("eq", 30);
+
+    cy.compareSnapshot({
+      name: "determinateCircular",
+      testThreshold: setThresholdBasedOnEnv(0.01),
+    });
+    cy.checkA11yWithWait();
+  });
+
+  it("renders lots of determinate with many sizes", () => {
+    mount(
+      <DeterminateCircularLoadingIndWithLabelSizeSmallDefaultLarge
+        progress={defaultArgs.progress}
+        max={defaultArgs.max}
+        min={defaultArgs.min}
+      />
+    );
+
+    cy.compareSnapshot({
+      name: "determinateCircularsWith3Sizes",
+      testThreshold: setThresholdBasedOnEnv(0.01),
+    });
+    cy.checkA11yWithWait();
+  });
+
+  it("renders determinate with icon", () => {
+    mount(
+      <DeterminateCirclularLoadingIndWithIcon
+        progress={defaultArgs.progress}
+        max={defaultArgs.max}
+        min={defaultArgs.min}
+      />
+    );
+
+    cy.get("ic-loading-indicator").invoke("prop", "size").should("eq", "icon");
+
+    cy.compareSnapshot({
+      name: "determinateCircularWithIcon",
+      testThreshold: setThresholdBasedOnEnv(0.01),
+    });
+    cy.checkA11yWithWait();
+  });
+
+  it("renders determinate linear", () => {
+    mount(
+      <DeterminateLinearLoadingIndWithLabel
+        progress={defaultArgs.progress}
+        max={defaultArgs.max}
+        min={defaultArgs.min}
+      />
+    );
+    cy.findShadowEl(
+      "ic-loading-indicator",
+      "ic-typography#ic-loading-label"
+    ).should(HAVE_TEXT, "Loading...");
+
+    cy.get("ic-loading-indicator").invoke("prop", "max").should("eq", 100);
+
+    cy.compareSnapshot({
+      name: "determinateLinear",
+      testThreshold: setThresholdBasedOnEnv(0.01),
+    });
+    cy.checkA11yWithWait();
+  });
+
+  it("renders lots of determinate linear with many sizes", () => {
+    mount(
+      <DeterminateLinearLoadingIndWithLabelSizeSmallDefaultLargeFull
+        progress={defaultArgs.progress}
+        max={defaultArgs.max}
+        min={defaultArgs.min}
+      />
+    );
+
+    cy.compareSnapshot({
+      name: "determinateLinearsWith3Sizes",
+      testThreshold: setThresholdBasedOnEnv(0.01),
+    });
+    cy.checkA11yWithWait();
+  });
+
+  it("renders determinate dark circular with label", () => {
```

Comment:
The test names should also be updated to say light instead of dark
```suggestion
  it("renders determinate light circular with label", () => {
```